staff/alejandro-perez/cv/styles.css

```diff
@@ -0,0 +1,36 @@
+p {
+	font-size: 1em;
+	line-height: 1.4em;
+	margin-bottom: 20px;
+    color: #444;
+    padding: 25px;
+    width: 700px;
+}
+
+#my-picture {
+    float: right;
+    width: 200px;
+    margin-right: 10%;
+}
+
+section,footer {
+    padding: 10px;
+}
+
+html {
+    background: linear-gradient(to bottom, #ffcc00 0%, #ff99cc 100%);
+}
+
+li {
+    line-height: 2em;
+}
+
+* {
+    font-family: 'Merriweather', serif;
+}
+
+h1:after{
+    display: block;
+    border: 2px solid black;
+    content: ' ';
+}
```

Comment:
You should follow a hierarchical order.

For example * and html should be on top

Reply:
Done!